ts/mains/main_node.ts

```diff
@@ -1023,6 +1023,60 @@ ipc.on('get-start-in-tray', event => {
   }
 });
 
+ipc.on('set-auto-start-enabled', (event, newValue) => {
+  try {
+    userConfig.set('autoStart', newValue);
```

Comment:
do we need to set/save this here? the setLoginItemSettings is not enough?

Reply:
yeah we can remove